runtime/platform.c

```diff
@@ -152,6 +152,12 @@ uintnat caml_mem_round_up_pages(uintnat size)
 static struct lf_skiplist mmap_blocks = {NULL};
 #endif
 
+Caml_inline void safe_munmap(uintnat addr, uintnat size)
+{
+  if (size > 0)
+    munmap((void*)addr, size);
+}
+
```

Comment:
Actually, you need to `#ifndef _WIN32` around this definition, otherwise Windows compilation fails, as demonstrated by Appveyor.  Or use a macro instead of an inline function.

Reply:
Oops - one day I'll set-up a local build doing both Unix and Windows at the same time 🤦

Gone with the `#ifdef` partly for "consistency" with the function below and mainly because the whole thing moves more neatly to unix.c in #11642